Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -24,6 +24,8 @@
 except Exception:
     pass
 
+CONTENT_RELEASE_VERSION = '0.0.0'
+CONTENT_BRANCH_NAME = 'master'
```

Comment:
I don't see any handling for branch name, should be in same place as update content release version

Reply:
@yaakovi What should be done here?

Reply:
replace `'master'` with current branch name.
branch name can be retrieved using:
```
branches = run_command('git branch')
branch_name_reg = re.search(r'\* (.*)', branches)
branch_name = branch_name_reg.group(1)
```